src/frontend/komponenter/Fagsak/Brevmottaker/LeggTilEndreBrevmottakerModal.tsx

```diff
@@ -73,12 +80,13 @@ export const LeggTilEndreBrevmottakerModal: React.FC = () => {
     const [navnErPreutfylt, settNavnErPreutfylt] = React.useState(false);
 
     React.useEffect(() => {
-        const navnSkalVærePreutfylt = erMottakerBruker(skjema.felter.mottaker.verdi);
-        if (navnSkalVærePreutfylt !== navnErPreutfylt) {
```

Comment:
Usikker på om jeg skal beholde en sånn her type sjekk, men for verge/fullmektig. F.eks hvis man fyller inn navn på verge, så bytter man til fullmektig, så vil navnet nå nullstilles 🤔

Reply:
La det inn